cogsmisc/customization.py

```diff
@@ -76,20 +76,15 @@ def __init__(self, bot):
         !spell Fly
         !monster Rat"
         """
+
         try:
             guild_prefix = self.bot.global_prefixes.get(ctx.message.server.id, self.bot.prefix)
         except:
             guild_prefix = self.bot.prefix
-        cmds = cmds.splitlines()
-        output = []
-        for cmd in cmds:
-            if cmd.startswith(guild_prefix) or cmd.startswith(self.bot.prefix) or not output:
-                output.append(cmd)
-            else:
-                output[-1] += "\n" + cmd
-        cmds = output
+        cmds = cmds.split(guild_prefix)
```

Comment:
This will cause unintended behaviour if the prefix is found anywhere in the command (e.g. `!`)